en/material-system/effect-2.x-to-3.0.md

```diff
@@ -0,0 +1,254 @@
+5. v3.0 also added the PipelineStates option, which is mainly used to define the pipeline states of material dependencies, such as DepthStencilState, BlendState, CullMode, etc.
+
+### 1.3 Editor Experience
+
+There are also some differences between v2.x and v3.0 in the editor experience of the material system.
+
+In v3.0, after selecting the node containing the model and material in the **Hierarchy** panel, the **Inspector** panel will display the component properties and the detailed Material configuration panel as follows.
+
+![effect](material-v3.png)
+
+In v2.x, when the node containing the model and material is selected in the **Node Tree** panel, the **Properties** panel will only show the component properties, not the detailed Material configuration panel:
+
+![effect](material-v2x.png)
+
+Instead, you need to jump to the **Assets** panel and select the Material asset before you can edit it in the **Properties** panel:
+
+![effect](material-panel-v2x.png)
+
+## 2. Effect Assets
+
+This section describes the commonalities and differences between assets in v2.x and v3.0.
+
+### 2.1 Effect format and content
+
+For the Effect asset, both v2.x and v3.0 use the YAML1.2 standard syntax and parser, and there are few differences between the two versions. The Effect asset defines a material type, and by following the [syntax format](yaml-101.md), the following information can be defined:
+
+- Technique list of rendering techniques
+- A list of passes for each Technique
+- A list of editable properties exposed to the editor in each Pass (including data type designation within the editor, component mapping relationships, etc.)
+- Shader programs for each Pass, including vertex and slice shader programs
+
+In terms of syntax details, such as Property declarations and macro definitions, the approach is the same:
+
+![effect](material4.jpg)
+
+![effect](material4.1.jpg)
+
+![effect](material5.jpg)
+
+### 2.2 Preset Material Types
+
+There is a big difference between v2.x and v3.0 in terms of preset materials.
+
+- The preset materials in v2.x include 2D Sprite, classic `blinn-phong` lighting materials, `unlit` materials, default `toon` materials, particle materials, etc.
+
+- The preset materials in v3.0 are physics-based rendering system, including standard PBR materials based on physical lighting, Skybox, toon rendering materials, 3D particle materials (CPU & GPU), particle trailing materials, traditional 2D Sprite materials, etc.
+
+The v3.0 default standard material supports the standard Physically Based Rendering (PBR) process, which contains a lot of mapping information to enhance the quality and realism of the material, such as diffuse mapping, normal mapping, metallic, roughness, ambient light oclusion, and so on. The whole algorithm is based on the standard BRDF lighting model, which is not available in v2.x. The overall performance of v3.0 is a step up from v2.x.
```

Comment:
diffuse map, normal map, metallic texture, roughness texture, ambient light occlusion texture, and so on
...
The overall rendering quality of v3.0 is also much more realistic than v2.x.